index.html

```diff
@@ -297,9 +297,7 @@ <h2>
                     provided media.</p>
                     <p>The provided media MUST include precisely one track of
                     each media type in <var>requestedMediaTypes</var>.
-                    The devices chosen MUST be the ones determined by the user.
-                    Once selected, the source of a
-                    {{MediaStreamTrack}} MUST NOT change.</p>
+                    The devices chosen MUST be the ones determined by the user.</p>
```

Comment:
Can we say something like MUST NOT CHANGE other than with the user's permission.

Reply:
PTAL at the new text.